contrib/runners/python_runner/python_runner.py

```diff
@@ -156,8 +161,15 @@ def run(self, action_parameters):
         LOG.debug('Setting env.')
         env = os.environ.copy()
         env['PATH'] = get_sandbox_path(virtualenv_path=virtualenv_path)
-        env['PYTHONPATH'] = get_sandbox_python_path(inherit_from_parent=True,
-                                                    inherit_parent_virtualenv=True)
+
+        sandbox_python_path = get_sandbox_python_path(inherit_from_parent=True,
+                                                      inherit_parent_virtualenv=True)
+        pack_common_libs_path = get_pack_common_libs_path(pack_db=pack_db)
+
+        if self._enable_common_pack_libs and pack_common_libs_path:
+            env['PYTHONPATH'] = pack_common_libs_path + ':' + sandbox_python_path
```

Comment:
Can you please also add some unit tests which assert on the actual `PYTHONPATH` for various scenarios (libs path is enabled, disabled, etc.)?

If they are already there, please ignore my comment, but I didn't see any. I only see `st2tests` test, but it only seems to be testing one code path (common libs enabled). Should be straight forward to add a unit test - Python runner action can just print PYTHONPATH and we can assert on that.

Reply:
Fixed in ce6fb89